ci/release.py

```diff
@@ -109,7 +109,7 @@ def update_version(args):
 
     run_cli(["git", "diff"])
     run_cli(["git", "add", str(changelog_path)])
-    run_cli(["git", "commit", "-m", f"Prepare release for {version_header}"])
+    run_cli(["git", "commit", "-m", f"[RELEASE] Prepare release for {version_header}"])
```

Comment:
Adding a `[RELEASE]` makes it more obvious that this commit will turn into the release. (And clearer to the CI)